container/shim/src/bin/shim.js

```diff
@@ -43,7 +47,7 @@ setTimeout(async function () {
     await register(true).catch((err) => {
       debug(`Failed to register ${err.name} ${err.message}`);
       // we don't try again if we fail the initial registration
```

Comment:
Do we want to try again now?

Reply:
Exit code was changed because we currently have `restart: unless-stopped` so I prefer the correct exit code.